packages/extension-base/src/background/types.ts

```diff
@@ -264,7 +264,7 @@ export interface RequestAccountList {
 export type RequestAccountSubscribe = null;
 
 export interface RequestActiveTabsUrlUpdate {
```

Comment:
```suggestion
export interface RequestActiveTabUrlUpdate {
```
now singular, right?